hashicorp/packer
# Backport of website: update dependencies into stable-website

## Backport

This PR is auto-generated from #12809 to be assessed for backporting due to the inclusion of the label backport/website.



The below text is copied from the body of the original PR.

---

### Description

Dev Portal was recently updated to use [Next V14](https://github.com/hashicorp/dev-portal/blob/main/package.json#L177). The package.json in the Packer website directory now has updated versions of next, node, npm, husky and other HashiCorp platform packages. Additionally, the Dockerfile has been updated to use a node v18 image. This is to prevent any errors during the deploy preview process and local development, which has been an issue for other product teams when developing locally & deploying changes to Dev Portal.

- [Asana task](https://app.asana.com/0/0/1206444672170571/f)

### Testing steps

- cd into `website` directory
- run `nvm use` -> should use node v18.27+ and npm v.9.6.7+
- run `npm install` -> should execute without errors
- run `npm run start` -> should execute without errors
- visit `localhost:3000` and it works as expected






---

<details>
<summary> Overview of commits </summary>

  - bc451baa46ea8af2121ac975899b8ec20e027266 

</details>
